pyanaconda/ui/lib/help.py

```diff
@@ -78,74 +126,73 @@ def _get_best_help_file(help_file):
     return None
 
 
-def get_help_path(help_file, plain_text=False):
-    """
-    Return the full path for the given help file name,
-    if the help file path does not exist a fallback path is returned.
-    There are actually two possible fallback paths that might be returned:
-
-    * first we try to return path to the main page of the installation guide
-      (if it exists)
-    * if we can't find the main page of the installation page, path to a
-      "no help found" placeholder bundled with Anaconda is returned
-
-    Regarding help l10n, we try to respect the current locale as defined by the
-    "LANG" environmental variable, but fallback to English if localized content
-    is not available.
-
-    :param help_file: help file name
-    :type help_file: str or NoneType
+def get_placeholder_path(plain_text=False):
+    """Get path to appropriate placeholder for given install class.
 
-    :param plain_text: should we find the help in plain text?
-    :type plain_text: bool
-
-    :return str: full path to the help file requested or to a placeholder
+    :param str plain_text: point to plain text version of the placeholder
+    :returns: path to placeholder or None if no placeholder is found
+    :rtype: str or None
     """
-    # help l10n handling
-
-    if help_file:
-        help_path = _get_best_help_file(help_file)
-        if help_path is not None:
-            return help_path
-
-    # setup the fallback files
     if plain_text:
-        main_page = HELP_MAIN_PAGE_TUI
```

Comment:
Please, remove the constants HELP_MAIN_PAGE_GUI and HELP_MAIN_PAGE_TUI.

Reply:
Removed.